protocol/v2/types/ssvshare.go

```diff
@@ -16,8 +16,17 @@ import (
 )
 
 const (
-	MaxPossibleShareSize = 1245
-	MaxAllowedShareSize  = MaxPossibleShareSize * 8 // Leaving some room for protocol updates and calculation mistakes.
+	// InitMaxPossibleShareSize - initial protocol max share size
+	InitMaxPossibleShareSize = 1245
+
+	// CurrentMaxPossibleShareSize - Current protocol version max share size;
+	// can be generated by generateMaxPossibleShare
+	// Used only for testing purposes
+	CurrentMaxPossibleShareSize = 1257
```

Comment:
i think we can delete these and instead just update the old value `MaxPossibleShareSize` to 1257